ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.14__periodics.yaml

```diff
@@ -97,6 +97,20 @@ tests:
           cpu: 100m
       timeout: 3h0m0s
     workflow: ipi-gcp
+- as: e2e-openstack
+  interval: 24h
```

Comment:
Do we need it every day? Do you expect changes to come from code we're not covering with presubmits?

Reply:
Correct. This was suggested from TRT.

Reply:
yeah good idea. let's make it run every week?

Reply:
I updated to weekly, let me know if that's ok.

Reply:
sorry I didn't see Joel's comment but IMO daily is too much... and we don't have that much of capacity to run it.
I'll suggest 72h or 1/week. Are you ok with it?

Reply:
What do you mean by capacity? IMO we should align it with the rest of the jobs which run daily to have a comparable signal. This is based on TRT's suggestion, which we followed.

Reply:
by capacity I mean our OpenStack clouds can't afford a bunch of daily CI jobs but I'll stop arguing here and TRT's suggestion, we'll see how it works in the future.